bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "echo-angular",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
No logging config is passed in from echo-angular service as it's the main app e.g. TN or whatever that uses this.  I can happily make this a new minor revision instead of a major - if someone moves to it the worst they will find is that logging level has changed due to them needing to tweak what the parent app passes in for config.